### DISCUSSION TITLE
How to Use

### DISCUSSION THREAD

**Discussion by changpenghui616** [memtest86plus/memtest86plus#93] [Q&A] [answered]
I have already build project on Linux system，  get memtest.efi and memtest.bin。
Now I want to run the memtest at uefi（EmulatorX64），but dont understand how to use the file

run memtest.efi on WinHost.exe，prompt issue
![image](https://user-images.githubusercontent.com/106718857/171601437-69e1ba05-dda0-4dd5-8f90-0ab9a4d51770.png)

**Comment by x86fr** [comment]
You must select the directory where `memtest.efi` is located first. 
Type `ls` to display the current directory content and `cd` to change directory.

**Reply by changpenghui616** [reply]
im try， but the issue still alive 。im cd FS0:，and 'Tab' can associate the memtest.efi
![image](https://user-images.githubusercontent.com/106718857/171606588-8f553d19-9f33-4c9c-8870-1c2abc55ced1.png)

![image](https://user-images.githubusercontent.com/106718857/171606660-ca76a1e6-d6db-43bb-a9a0-670dc91173cb.png)

And，WinHost output some message，i think the memtest.efi already be used to EmulatorX64， but don't understand the error in further  
![image](https://user-images.githubusercontent.com/106718857/171608784-15bc6f44-05f3-4a20-babc-8c6b5e215aa3.png)

**Comment by x86fr** [comment]
please try to run :
`ls memtest.efi`
then
`.\memtest.efi`
And post a screenshot

If this doesn't work, rename `memtest.efi` to `bootx64.efi` and put it in the directory `\EFI\Boot\`

**Reply by changpenghui616** [reply]
![image](https://user-images.githubusercontent.com/106718857/171623107-5d89b07e-1106-43e5-a9dc-27faea0c5529.png)
![image](https://user-images.githubusercontent.com/106718857/171623117-872f7b2e-8043-470c-9580-c194510ed29f.png)

look still doesn't work

**Comment by x86fr** [comment]
Is `HelloWorld.efi` working?

I'm not sure how your emulator is working and even if it's possible to run any software like memtest86+ on it.

**Reply by changpenghui616** [reply]
HelloWorld.efi can work ，and output the normal message.
the emulator from EDK2.
I try to run HelloWorld.efi and memtest.efi in real PC machine（HONOR V14），operate same（rename to bootx64.efi and put it in EFI\boot ），HelloWorld.efi can work，memtest.efi doesn't work still（black screen）

**Reply by x86fr** [reply]
Can you try to use the official tool on memtest.org to create a bootable USB drive and check if it works?

**Reply by changpenghui616** [reply]
okay ，i will try to test this way

**Reply by changpenghui616** [reply]
use mt86plus_6.00b2_USB_Installer.exe and create a bootable USB drive ，it seems doesn't work still， no information or error， just black screen！
but use other PC machine , it can work（Honor magicbook 14 2021， and memory type is DDR4）
PS：my PC memory type is LPDDR5. whether memtestx86 unsupport it now

**Comment by x86fr** [comment, marked as answer]
Please wait some days, I think I know what's the issue with your Honor V14 laptop.

**Reply by changpenghui616** [reply]
thanks, Looking forward to your replay

**Comment by changpenghui616** [comment]
whether find any issue about this error？

**Reply by x86fr** [reply]
please try this the latest night build (available here: https://memtest.org/dev) on you Honor laptop

**Reply by changpenghui616** [reply]
Good news，it can work on my PC（https://memtest.org/dev）
thanks.
can i know whats the problem？

**Reply by x86fr** [reply]
It was related to #102

**Reply by changpenghui616** [reply]
thanks
